codice/alliance
# CAL-131: Add all Nitf attributes into Nitf metacard types

#### What does this PR do?
- Adds all previously available NITF attributes (and current supported TREs fields) into both Image and GMTI NITF `MetacardType`s under the `ext.nitf.*` namespace.
- NITF Attributes that also map to the taxonomy are duplicated in the `Metacard`. This is so that Imagery Analysts can operate on the `Metacard`'s attributes with terms they are familiar with.
- Removes `MtirpbAttribute.AIRCRAFT_LOCATION` -> `Isr.DWELL_LOCATION` taxonomy mapping.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@dcruver @lcrosenbu @bdeining @glenhein 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith 
@pklinef
#### How should this be tested?
Full build
Start an instance of Alliance and ingest some NITFs, verify that attributes are visible in the `Metacard` view in the UI as well as in the `Metacard` XML representation.

#### Any background context you want to provide?
Some NITF specific fields that Imagery Analysts want, in order to perform operations against, do not map to the taxonomy. The decision was made that all fields of a NITF will be added as `Attribute`s under the `ext.nitf.*` namespace. This way Imagery Analysts can operate on NITF specific fields that do not map to the taxonomy. NITF fields that map to the taxonomy will be duplicated as one of the taxonomy attributes, and in an `Attribute` under `ext.nitf.*`.
#### What are the relevant tickets?
[CAL-131](https://codice.atlassian.net/browse/CAL-131)
#### Screenshots (if appropriate)
#### Checklist:
